kobuki/kobuki.bugzoo.yml

```diff
@@ -11,6 +11,16 @@ blueprints:
       UBUNTU_VERSION: xenial
       ROS_DISTRO: kinetic
       CATKIN_PKG: kobuki_ftdi
+  - type: docker
+    tag: robustrosin/robust:b166c93
+    file: ../Dockerfile
+    context: b166c93
+    arguments:
+      IS_BUILD_FAILURE: no
```

Comment:
Not sure about this yet.

Reply:
Isn't this really a question to @ChrisTimperley ?

Reply:
I don't see your name at-mentioned :)